.github/workflows/job_python_unit_tests.yml

```diff
@@ -246,15 +246,6 @@ jobs:
           TEST_PRECISION: FP32
           PYTORCH_TRACING_MODE: TORCHFX
 
-      - name: PyTorch torch.compile TORCHSCRIPT Layer Tests
-        if: ${{ fromJSON(inputs.affected-components).PyTorch_FE.test && runner.os != 'macOS' && runner.arch != 'ARM64' }} # Ticket: 126287
-        run: |
-          python3 -m pytest ${LAYER_TESTS_INSTALL_DIR}/pytorch_tests -m precommit_ts_backend --junitxml=${INSTALL_TEST_DIR}/TEST-pytorch.xml
-        env:
-          TEST_DEVICE: CPU
-          TEST_PRECISION: FP32
-          PYTORCH_TRACING_MODE: TORCHSCRIPT
-
```

Comment:
why is it removed?

Reply:
This is an experimental backend that we created when we just started working on torch fx backend. Now that fx backend is mature we no longer need this backend and haven't been testing any models on it for a while now. So we are removing it.

Reply:
Thanks maxim. Removed.